sdk/logs/src/main/java/io/opentelemetry/sdk/logs/SdkLogRecordBuilder.java

```diff
@@ -121,25 +125,30 @@ public void emit() {
       return;
     }
     Context context = this.context == null ? Context.current() : this.context;
+    if (!logger.isEnabled(severity, context)) {
+      return;
+    }
     long observedTimestampEpochNanos =
         this.observedTimestampEpochNanos == 0
             ? this.loggerSharedState.getClock().now()
             : this.observedTimestampEpochNanos;
     loggerSharedState
         .getLogRecordProcessor()
-        .onEmit(
-            context,
-            SdkReadWriteLogRecord.create(
-                loggerSharedState.getLogLimits(),
-                loggerSharedState.getResource(),
-                instrumentationScopeInfo,
-                timestampEpochNanos,
-                observedTimestampEpochNanos,
-                Span.fromContext(context).getSpanContext(),
-                severity,
-                severityText,
-                body,
-                attributes,
-                eventName));
+        .onEmit(context, createLogRecord(context, observedTimestampEpochNanos));
+  }
+
+  protected ReadWriteLogRecord createLogRecord(Context context, long observedTimestampEpochNanos) {
```

Comment:
extracted this so don't need to add

```
if (!logger.isEnabled(severity, context)) {
  return;
}
```

to ExtendedSdkLogRecordBuilder

(you can thank codecov for pointing out to me that there was no test coverage for it in the subclass 😄)